README.md

```diff
@@ -3,17 +3,24 @@ appium-chromedriver
 
 [![Release](https://github.com/appium/appium-chromedriver/actions/workflows/publish.js.yml/badge.svg)](https://github.com/appium/appium-chromedriver/actions/workflows/publish.js.yml)
 
-Node.js wrapper around [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/).
+Node.js wrapper around [Chromedriver](https://sites.google.com/chromium.org/driver/)
+and [Microsoft Edge WebDriver](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/).
 This wrapper is not used directly in Appium, but rather by various Android drivers to automate
 Chrome/Chromium-based browsers
 and web views using Hybrid Mode approach. Check the corresponding driver tutorials to get
 more details on it.
 
 > **Note**
 >
-> The normal use of this package is via an Appium driver such as [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver/) and not directly.
+> The normal use of this package is via an Appium driver such as [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver/) and
+> [appium-chromium-driver](https://github.com/appium/appium-chromium-driver) and not directly.
```

Comment:
This package is intended to be used as a helper module for Appium drivers such as ... It was not created for standalone usage.